src/components/project-card.tsx

```diff
@@ -0,0 +1,97 @@
+"use client";
+
+import { useDictionary } from "@/contexts/dictionary-provider";
+import Image from "next/image";
+import AppLink from "./link";
+
+interface ProjectProps {
+  type: string;
+}
+
+const ProjectCard = ({ type }: ProjectProps) => {
+  const dict = useDictionary();
+
+  const info = () => {
+    switch (type) {
+      case "sei":
+        return {
+          src: "/logo/sei.svg",
+          alt: "SEI Logo",
+          gradient_color: "rgba(35,11,183,0.10)",
+          ref: "https://seium.org/",
+        };
+      case "bugsbyte":
+        return {
+          src: "/logo/bugsbyte.svg",
+          alt: "BugsByte Logo",
+          gradient_color: "rgba(120,176,69,0.25)",
+          ref: "https://bugsbyte.org/",
+        };
+      case "coderdojo":
+        return {
+          src: "/logo/coderdojo.svg",
+          alt: "CoderDojo Logo",
+          gradient_color: "rgba(50,51,51,0.25)",
+          ref: "https://coderdojobraga.org/",
+        };
+      default:
+        return {
+          src: "/logo/sei.svg",
+          alt: "SEI Logo",
+          gradient_color: "rgba(35,11,183,0.10)",
+          ref: "https://seium.org/",
+        };
+    }
+  };
+
+  const getCardIndex = () => {
+    switch (type) {
+      case "sei":
+        return 0;
+      case "bugsbyte":
+        return 1;
+      case "coderdojo":
+        return 2;
+      default:
+        return 0;
+    }
+  };
+
+  const cardIndex = getCardIndex();
+  const project = dict.about.projects.card[cardIndex];
+
+  if (!project) return null;
+
+  return (
+    <div
+      className={`mt-[30px] border-b border-[#230BB71A] pb-[30px] md:mt-0 md:min-w-[460px] lg:w-1/3 lg:border-none`}
```

Comment:
```suggestion
      className={`mt-[30px] border-b border-[#230BB71A] pb-[30px] md:mt-0 md:min-w-[460px] md:w-1/3 md:border-none`}
```

This border should only be visible when on mobile mode, there was a point between lg and md where it was visible even though it was in desktop mode, this should fix it :)